apiary-metastore-events/README.md

```diff
@@ -4,9 +4,10 @@
 Apiary Metastore Events contains a set of modules responsible for retrieving and processing Hive Metastore events.
 
 Currently, the following modules are defined:
- - [apiary-metastore-listener](apiary-metastore-listener) - listens to events from the Hive Metastore and publishes them to an SNS queue.
- - [apiary-receivers](apiary-receivers) - provides an implementation for receiving and deserializing Hive events from a queue.
- 
+ - [apiary-metastore-listener](apiary-metastore-listener) - listens to events from the Hive Metastore and publishes them to an SNS queue
```

Comment:
publishes them to a "SNS topic"

Reply:
I will update that description with the next commit. @patduin suggested to update it to: 
Consumers that receive and process Hive Metastore event messages published by the `apiary-metastore-listener`.